src/common/AZERITE_SCALING.generated.json

```diff
@@ -1390,13 +1390,13 @@
     "scaling_type": -7,
     "effects": {
       "725525": {
-        "avg": 0
+        "avg": 2.11156
       },
       "725608": {
-        "avg": 0
+        "avg": 2.11156
       },
       "725609": {
-        "avg": 0
+        "avg": 2.11156
```

Comment:
like I said in the PR description, ignore this, just to make it display all 4 stats for demo purpose for now.